app/frontend/src/components/forms/submission/MySubmissionsActions.vue

```diff
@@ -103,6 +103,10 @@ export default {
     formId: {
       type: String,
       required: true
+    },
+    isCopyFromExistingSubmissionEnabled: {
```

Comment:
This shouldn't be a prop in the front end, there should be a getter/setter from the form store which uses a service that gets/sets the setting in the backend. Look at `enableSubmitterDraft`